### DISCUSSION TITLE
[单独展示页] 脱离博客模板，仅分享一个notion页面

### DISCUSSION THREAD

**Discussion by zephyr-y** [tangly1024/NotionNext#2074] [Ideas]
**为什么提出这个新的特性改动**
很多时候，不需要模板框架，只需要简单的展示一个单独的页面

**描述一下你推荐的解决方案**
不要模板框架，只是把notion里的文章内容展示出来，可以再加一个评论

**描述一下你考虑过的其它替代解决方案**
目前没看到可选的方案

**补充说明**
纯展现一个单独页面。这个需求应该还是挺普遍的

**Comment by tangly1024** [comment]
脱离模板的话，建议的方案

直接将notion页面分享给别人，然后底部套上一些评论插件：

类似这个工具：
https://www.notioncomments.com/

如果您希望分享的notion页面可以用自定义域名，则可以尝试fruition方案：

https://www.tangly1024.com/article/fruition-notion
